web-ui/src/pages/FeedbackRequestPage.jsx

```diff
@@ -123,9 +129,19 @@ const FeedbackRequestPage = () => {
     }
   }, [activeStep, hasFor, hasTemplate, hasFrom, hasSend, dueQuery, isValidDate]);
 
-  const handleSubmit = useCallback(() => {
-    history.push("/feedback/request/confirmation");
-  }, [history]);
+const handleSubmit = () =>{
+    let feedbackRequest = {}
+    if (typeof fromQuery === 'string' && fromQuery.indexOf(',') === -1) {
+        feedbackRequest = { id : null, creatorId: currentUserId, requesteeId:forQuery, recipientId: fromQuery, templateId:"6b72840f-7e18-43cc-a923-15dec8ef77f4", sendDate: sendDate, dueDate: dueQuery, status: "Pending", submitDate: null}
+        sendFeedbackRequest(feedbackRequest)
+    } else if (typeof fromQuery === 'string' && fromQuery.indexOf(',') !== -1) {
+      let fromArray = fromQuery.split(',')
+        for (const recipient of fromArray) {
+           feedbackRequest = { id : null, creatorId: currentUserId, requesteeId: forQuery, recipientId: recipient, templateId: "6b72840f-7e18-43cc-a923-15dec8ef77f4", sendDate: sendDate, dueDate: dueQuery, status: "Pending", submitDate: null}
```

Comment:
This can be simplified since `.split(",")` will still work on a string with no commas.

Reply:
Resolved